src/js/components/DataFilters/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { BoxProps } from '../Box/index';
+
+export interface DataFiltersProps {
+  drop?: boolean;
+  layer?: boolean;
+  search?: boolean;
```

Comment:
I don't see search as a prop in DataFilters. `heading` and `children` should be included

Reply:
added